kolibri_instant_schools_plugin/kolibri_plugin.py

```diff
@@ -20,6 +20,8 @@ class User(KolibriPluginBase):
     root_view_urls = "root_urls"
     translated_view_urls = "urls"
     untranslated_view_urls = "untranslated_urls"
+    django_settings = "instant_schools_settings"
```

Comment:
Is this something built-in to the KolibriPluginBase class, or something Django-related (or both)?

Reply:
The former!